test/services/optout/OptOutServiceNextUpdatesPageOptOutViewChecksSpec.scala

```diff
@@ -125,9 +125,9 @@ class OptOutServiceNextUpdatesPageOptOutViewChecksSpec extends UnitSpec
           ITSAStatusUpdateResponseSuccess()
         ))
 
-        val result = service.nextUpdatesPageOptOutChecks()
+        val result = service.nextUpdatesPageChecksAndProposition()
 
-        result.futureValue shouldBe a[NextUpdatesQuarterlyReportingContentChecks]
```

Comment:
guessing same here, the method returns a tuple of two values, I assume we want to check both contain the  expected values.